docs/UserGuide.md

```diff
@@ -850,68 +903,64 @@ Examples:
 
 Generates a referral letter for a patient in PDF format.
 
-Format: `referral ti/TITLE di/DOCTOR_ID pi/PATIENT_ID [d/DESCRIPTION]`
+**Format**: `referral ti/TITLE di/DOCTOR_ID pi/PATIENT_ID [d/DESCRIPTION]`
 
 The parameters are:
 
-* `ti/TITLE` title of the referral document.
-* `di/DOCTOR_ID` id of the doctor to be referred to.
-* `pi/PATIENT_ID` id of the patient being referred.
-* `d/DESCRIPTION` description of the patient's condition and further details.
+Parameters      |  Explanation                                                    | Constraints                                                                     |                
+----------------|-----------------------------------------------------------------|---------------------------------------------------------------------------------|
+`ti/TITLE`      | the title of the referral document.                             | Must only contain alphanumeric characters and spaces, and it should not be blank|
+`di/DOCTOR_ID`  | id of the doctor to be referred to.                             | Must be in the form of `DXXX` / `dXXX` where `XXX` is 3-digit integer. For the full information, please refer to [this](#overview)|
+`pi/PATIENT_ID` | id of the patient being referred.                               | Must be in the form of `PXXX` / `pXXX` where `XXX` is 3-digit integer. For full info, please refer to [this](#overview)|
+`d/DESCRIPTION` | description of the patient's condition and further details.     | Must only contain alphanumeric characters and spaces, and it should not be blank|
 
-Note: 
+**Note:**
 
-* the patient and doctor must both be present in the GoMedic App as a valid entry or the referral will not be
-generated.
+the patient and doctor must both be present in the GoMedic App as a valid entry or the referral will not be generated.
 
```

Comment:
I think it would be good to mention that the title is used as the name of the pdf also, besides the title of the referral document, again you can use the alert box for this